bin/package.json

```diff
@@ -1,11 +1,23 @@
 {
   "name": "@glimmer-workspace/bin",
+  "type": "module",
   "private": true,
+  "repo-meta": {
+    "strictness": "loose",
+    "env": [
+      "node",
+      "console"
+    ],
+    "lint": [
+      "*"
+    ]
+  },
   "scripts": {
-    "test:lint": "eslint .",
+    "test:lint": "eslint . --quiet",
```

Comment:
This PR uses `warning` level lints for situations where we want to annotate a line with a `disable`
as documentation for the future (when we enable stricter lints), but don't want the lint to fail
yet. It's a temporary measure, and we should switch back to error-only lints once we've moved
everything to stricter lints.